src/index.css

```diff
@@ -1,27 +1,72 @@
-body {
+*,
+*::before,
+::after {
   margin: 0;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    'Segoe UI',
-    'Roboto',
-    'Oxygen',
-    'Ubuntu',
-    'Cantarell',
-    'Fira Sans',
-    'Droid Sans',
-    'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+  padding: 0;
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+#root {
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.containerCalculator {
+  width: 80%;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+}
```

Comment:
- [ ] On desktop screens the calculator is stretching and not looking good, Please make sure you fix this issue.
   - To fix this issue. You should give it a fixed width in `pixels` not in `%` ex: `width: 500px;`.

![image](https://user-images.githubusercontent.com/83803414/213194430-a1c4b88f-8d65-4f2e-979c-b8027df89125.png)

Reply:
thanks for the thanks for the support any suggestions 😄